src/page/stylesheets/reset.scss

```diff
@@ -490,7 +490,7 @@
   font-family: arial, helvetica, sans-serif;
   font-size: small;
   background: white;
-  -webkit-appearance: push-button;
+  appearance: button; /* [Deprecation] The keyword 'push-button' specified to an 'appearance' property is not standardized. It will be removed in the future. */
```

Comment:
We don't need this comment if we are no longer using `push-button` now.

Reply:
Done and Done!